src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,124 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String createManufacturerQuery = "INSERT INTO manufacturers (name, country) VALUES (?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createManufacturerStatement =
+                        connection.prepareStatement(createManufacturerQuery,
+                                Statement.RETURN_GENERATED_KEYS)) {
+            createManufacturerStatement.setString(1,
+                    manufacturer.getName());
+            createManufacturerStatement.setString(2,
+                    manufacturer.getCountry());
+            createManufacturerStatement.executeUpdate();
+            ResultSet generatedKeys = createManufacturerStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't can't insert manufacturer "
+                    + manufacturer + " to DB", e);
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        Optional<Manufacturer> result = Optional.empty();
+        String getManufacturerQuery = "SELECT * FROM manufacturers "
+                + "WHERE is_deleted = false AND id = ?;";

```

Comment:
All the keywords should be uppercased in sql 
```suggestion
                + "WHERE is_deleted = FALSE AND id = ?;";

```